ocaml/xapi/cli_frontend.ml

```diff
@@ -2631,7 +2631,23 @@ add a mapping of 'path' -> '/tmp', the command line should contain the argument
 			help="forget a PVS farm";
 			implementation=No_fd Cli_operations.PVS_farm.forget;
 			flags=[];
-		}
+		};
+		"pvs-server-introduce",
+		{
+			reqd=["addresses"; "first-port"; "last-port"; "pvs-farm-uuid"];
```

Comment:
Will change it. I wanted to avoid ambiguity if other farms existed (or could exist).